lib/routes/admin/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+module.exports = function(mediator) {
+  const router = express.Router();
```

Comment:
We still have to support node 10. Can you use `var` for the moment.

Might also want to call it `adminRouter` to be a bit more specific.